README.md

```diff
@@ -45,12 +45,21 @@ Try out this template repository using Codespaces following these steps:
 
 Next, we will run our app.
 
+
+
 ### Inspect your codespaces environment
 
 What you have at this point is a pre-configured environment where all the runtimes and libraries you need are already installed - a zero config experience.
 
 > This environment will run the same regardless of whether your students are on Windows, macOS or Linux.
 
+
+
+
+<details> 
+
+<summary><b>Run API</b></summary>
```

Comment:
This `summary` is repeating the same title is collapsing so when expanding it looks repeated. Can you remove the title below from the collapsed sections?